ScrubBar/ScrubBar.swift

```diff
@@ -1,38 +1,62 @@
 import UIKit
 
+public protocol ScrubBarDelegate: class {
+
+    func scrubBar(_ scrubBar: ScrubBar, didSelectItemAt selectedIndex: Int)
+
+}
+
 public class ScrubBar: UIControl {
 
     // MARK: Initialization
 
-    public init(images: [UIImage]) {
-        imageViews = images.map { UIImageView(image: $0) }
+    public init?(items: [ScrubBarItem]) {
+        guard items.count > 0 else { return nil }
+
+        self.items = items
+        imageViews = items.map { UIImageView(image: $0.image) }
         super.init(frame: .zero)
+
         configureImageViews()
```

Comment:
Consider extracting `didInit()` with for the following calls:

```swift
configureImageViews()
configureStackView()
configureSelectionView()
```

Once done, consider removing unnecessary newlines in `init?`.